kyuubi-hive-beeline/src/main/resources/BeeLine.properties

```diff
@@ -154,73 +154,76 @@ hs2-unknown-connection-problem: Unknown HS2 problem when communicating with Thri
-\  --showWarnings=[true/false]     display connection warnings\n \
-\  --showDbInPrompt=[true/false]   display the current database name in the prompt\n \
-\  --showNestedErrs=[true/false]   display nested errors\n \
-\  --numberFormat=[pattern]        format numbers using DecimalFormat pattern\n \
-\  --force=[true/false]            continue running script even after errors\n \
-\  --maxWidth=MAXWIDTH             the maximum width of the terminal\n \
-\  --maxColumnWidth=MAXCOLWIDTH    the maximum width to use when displaying columns\n \
-\  --silent=[true/false]           be more silent\n \
-\  --autosave=[true/false]         automatically save preferences\n \
-\  --outputformat=[table/vertical/csv2/tsv2/dsv/csv/tsv/json/jsonfile]  format mode for result display\n \
-\                                  Note that csv, and tsv are deprecated - use csv2, tsv2 instead\n \
-\  --incremental=[true/false]      Defaults to false. When set to false, the entire result set\n \
-\                                  is fetched and buffered before being displayed, yielding optimal\n \
-\                                  display column sizing. When set to true, result rows are displayed\n \
-\                                  immediately as they are fetched, yielding lower latency and\n \
-\                                  memory usage at the price of extra display column padding.\n \
-\                                  Setting --incremental=true is recommended if you encounter an OutOfMemory\n \
-\                                  on the client side (due to the fetched result set size being large).\n \
-\                                  Only applicable if --outputformat=table.\n \
-\  --incrementalBufferRows=NUMROWS the number of rows to buffer when printing rows on stdout,\n \
-\                                  defaults to 1000; only applicable if --incremental=true\n \
-\                                  and --outputformat=table\n \
-\  --truncateTable=[true/false]    truncate table column when it exceeds length\n \
-\  --delimiterForDSV=DELIMITER     specify the delimiter for delimiter-separated values output format (default: |)\n \
-\  --isolation=LEVEL               set the transaction isolation level\n \
-\  --nullemptystring=[true/false]  set to true to get historic behavior of printing null as empty string\n \
-\  --maxHistoryRows=MAXHISTORYROWS The maximum number of rows to store beeline history.\n \
-\  --delimiter=DELIMITER           set the query delimiter; multi-char delimiters are allowed, but quotation\n \
-\                                  marks, slashes, and -- are not allowed; defaults to ;\n \
-\  --convertBinaryArrayToString=[true/false]    display binary column data as string or as byte array \n \
-\  --help                          display this message\n \
+cmd-usage: Usage: kyuubi-beeline <options>. \n\n\
+Options:\n \
+\  -u <database url>                 The JDBC URL to connect to.\n \
```

Comment:
could you please keep the spaces? I don't see much benefit with extra two spaces

Reply:
Yeah. I will remove these two extra spaces in the next commit.